packages/preview-middleware-client/src/adp/init.ts

```diff
@@ -8,16 +8,14 @@ import {
     startPostMessageCommunication,
     enableTelemetry
 } from '@sap-ux-private/control-property-editor-common';
-
 import { ActionHandler } from '../cpe/types';
-import VersionInfo from 'sap/ui/VersionInfo';
 import { getUI5VersionValidationMessage } from './ui5-version-utils';
 import UI5Element from 'sap/ui/dt/Element';
 import { getError } from '../cpe/error-utils';
-import type {SingleVersionInfo} from '../../types/global';
+import { getUi5Version } from '../utils';
```

Comment:
This will result in runtime crash.

UI5 module loader can't handle implicit index files, so we need to explicitly add it in the module name or avoid the indirect import entirely.

```suggestion
import { getUi5Version } from '../utils/version';
```

Reply:
So basically you say what @nikmace suggested will not work, right? Ok then let's revert it 👍🏻